### ISSUE TITLE
PerformanceServerTiming entry always has a duration field even when the original header did not

### ISSUE THREAD

**Issue by andydavies** [w3c/server-timing#95]
The PerformanceServerTiming entry always has a duration field even if the original Server-Timing header did not

For example, if the response contains this header

```
Server-Timing: cdn-cache; desc=REVALIDATE
```

Then the corresponding entry will be:

```
PerformanceServerTiming
  description: "REVALIDATE"
  duration: 0
  name: "cdn-cache"
```

This makes is impossible to know just from the API whether the header had no duration, or whether it had a duration and it was 0

**Comment by andydavies** [comment]
Yeh, `duration` being nullable would work for our use case.

I'll have a think about the invalid value case and I'm not sure on the answer ATM. I think null would be my preference but will ask the rest of the team

**Comment by nicjansma** [comment]
Agreed that a nullable `duration` would be ideal here so we can distinguish.

Personally I think if the number can't be parsed, `duration` being set to `null` would be fine as well.
